account-info/src/admins.rs

```diff
@@ -0,0 +1,74 @@
+use contract::{contract_api::{runtime, storage}, unwrap_or_revert::UnwrapOrRevert};
+use types::{Key, URef, account::AccountHash};
+
+use crate::utils;
+
+use super::ContractError;
+
+pub const ADMINS_DICT: &str = "admins";
+pub const ADMINS_COUNT: &str = "admins_count";
+pub const ADMIN_ACTIVE: bool = true;
+pub const ADMIN_DISABLED: bool = false;
+
+pub struct Admins {
+    dict_uref: URef
+}
+
+impl Admins {
+    pub fn new() -> Admins {
+        let dict_key: Key = runtime::get_key(ADMINS_DICT).unwrap_or_revert();
+        let dict_uref: &URef = dict_key.as_uref().unwrap_or_revert();
+        Admins {
+            dict_uref: *dict_uref
+        }
+    }
+
+    pub fn is_admin(&self, address: &AccountHash) -> bool {
+        let result: Option<bool> = storage::dictionary_get(self.dict_uref, &address.to_string())
+            .unwrap_or_revert();
+        match result {
+            Some(value) => value == ADMIN_ACTIVE,
+            None => false
+        }
+    }
+
+    pub fn add(&self, address: &AccountHash) {
+        if self.is_admin(address) {
+            runtime::revert(ContractError::AdminExists);
+        } else {
+            storage::dictionary_put(self.dict_uref, &address.to_string(), ADMIN_ACTIVE);
+            
+            // Increment the admin count.
+            let admins_count: u32 = utils::get_key(ADMINS_COUNT);
+            utils::set_key(ADMINS_COUNT, admins_count + 1);
```

Comment:
Could you please confirm that the deploys are executed one by one. I remember that Ed said it is, but want to double check that there was no misunderstanding on my side. Thanks!